packages/@azure/servicebus/data-plane/test/README.md

```diff
@@ -25,9 +25,34 @@
   unpartitioned-topic-subscription | Subscription with sessions disabled in the Topic, `unpartitioned-topic` | SUBSCRIPTION_NAME_NO_PARTITION
   partitioned-topic-sessions-subscription | Subscription with sessions enabled in the Topic, `partitioned-topic-sessions` | SUBSCRIPTION_NAME_SESSION
   unpartitioned-topic-sessions-subscription | Subscription with sessions enabled in the Topic, `unpartitioned-topic-sessions` | SUBSCRIPTION_NAME_NO_PARTITION_SESSION
-  topic-filter | Topic for testing topic filters | TOPIC_FILTER
-  topic-filter-subscription | Subscription in the Topic `topic-filter` | TOPIC_FILTER_SUBSCRIPTION
-  topic-filter-default-subscription | Subscription in the Topic `topic-filter` | TOPIC_FILTER_DEFAULT_SUBSCRIPTION
+  topic-filter | Topic for testing topic filters | TOPIC_FILTER>NAME
+  topic-filter-subscription | Subscription in the Topic `topic-filter` | TOPIC_FILTER_SUBSCRIPTION_NAME
+  topic-filter-default-subscription | Subscription in the Topic `topic-filter` | TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME
+
+
+    Following is a sample .env file template that can be re-used for your environment:
```

Comment:
If we speak of .env file, then we should also talk about the usage of the `.dotenv` module.

Reply:
Right, the instructions on how to set these variables are missing as well. Will update.

Reply:
Ensure that you have a note that having a `.env` file is optional
They can set the environment variable manually or name their entities with the default names

Reply:
Do check the latest file, hoping the language conveys it fine